packages/shared/src/utils/__test__/dataURLtoFile.test.ts

```diff
@@ -0,0 +1,21 @@
+import { dataURLtoFile } from 'shared/utils';
+
+describe('dataURLtoFile', () => {
+  test('이미지 파일 변환 테스트', () => {
```

Comment:
테스트 설명을 결과와 연관지어 작성해주세요.
인자로 전달한 데이터와 assertion(ex. toBe, toBeInTheDocument)을 연관지으면 이렇게 작성할 수 있을 것 같습니다.

```suggestion
test('PNG Data URL을 변환하면 올바른 파일명과 MIME 타입을 가진 File 객체가 반환되어야 한다.', () => {
  const dataURL = 'data:image/png;base64,iVBORw0KGgo=';
  const filename = 'test.png';
  const file = dataURLtoFile(dataURL, filename);
  
  expect(file.name).toBe('test.png');
  expect(file.type).toBe('image/png');
});